src/Workspaces/Core/Portable/ExternalAccess/UnitTesting/Api/UnitTestingSymbolKeyExtensions.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting.Api
+{
+    internal static class UnitTestingSymbolKeyExtensions
+    {
+        public static string UnitTesting_ToString(this SymbolKey symbolKey)
+            => symbolKey.ToString();
```

Comment:
😕 Why is this needed? `ToString` is a public method on all types.

Reply:
I have removed this part and put it into the UnitTestingSymbolKeyWrapper